### PR TITLE
css-three-column-layout-Dylan De Vei

### PR DESCRIPTION
**Pull request by dylandevei**
Screen Shot 2022-03-17 at 10 59 05 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Dylan! your circles are too large as compared to the example, please revise and submit a new screenshot with your revision

**Comment by dylandevei** [comment]
Screen Shot 2022-03-18 at 11 58 21 AM
Revisions for the challenge (hope they are right!)

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+.row {
+  display: flex;
+  margin: auto;
+}
+
+.column-full {
+  max-width: 100%;
+  text-align: center;
+  margin: auto;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css]: The only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+.column-one-third {
+  text-align: center;
+  width: calc(100% / 3);
+  padding: 10px 70px;
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css]: The only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+section > p {
+  font-family: Lato, serif;
+  font-style: normal;
+  font-weight: 400;
+  color: lightgray;
+  text-align: center;
+}
+}
```
Comment (codingcodymiller) [inline css-three-column-layout/styles.css]: You have an extra curly brace here 👀

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by dylandevei** [comment]
Screen Shot 2022-03-18 at 5 34 31 PM
Revisions (progress).

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**